src/components/v2/MintRepayVai/Mint/index.tsx

```diff
@@ -0,0 +1,127 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import BigNumber from 'bignumber.js';
+import Typography from '@mui/material/Typography';
+
+import { convertWeiToCoins } from 'utilities/common';
+import { Icon } from '../../Icon';
+import { TokenTextField } from '../../TokenTextField';
+import { SecondaryButton } from '../../Button';
+import { useStyles } from './styles';
+
+const VAI_SYMBOL = 'vai';
+
+export interface IMintUiProps {
+  disabled: boolean;
+  isSubmitting: boolean;
+  onSubmit: (value: BigNumber) => Promise<void>;
+  limitWei: BigNumber;
+  mintFeePercentage: number;
+}
+
+// TODO: Move to dashboard component/container once created
+export const MintUi: React.FC<IMintUiProps> = ({
```

Comment:
This could be a nice naming convention to follow if the presentational component and container are in the same file and we want the container to have the final component name

Reply:
I hesitated on that to be honest. If you like it, I'm cool with following that pattern.